docs/tasks/debug-application-cluster/logging-stackdriver.md

```diff
@@ -80,7 +80,7 @@ Stackdriver Logging agents to the running cluster.
 1. Deploy a `ConfigMap` with the logging agent configuration by running the following command:
 
     ```
-    kubectl create -f https://k8s.io/docs/tasks/debug-application-cluster/fluentd-gcp-configmap.yaml
+    kubectl create --namespace=kube-system -f https://k8s.io/docs/tasks/debug-application-cluster/fluentd-gcp-configmap.yaml
     ```
 
     The command creates the `ConfigMap` in the `default` namespace. You can download the file
```

Comment:
please update this text (per @steveperry-53 suggestion) to match the code change you're making